patterns/implies_presence_of.yaml

```diff
@@ -17,6 +17,11 @@ annotations:
   - annotationProperty: implies_presence_of_some
     var: anatomical_entity
 
+name:
+  text: "%s presence"
```

Comment:
Any reason you don't want to keep this consistent with absence (which is `%s absent` not `%s absence`). Also, wouldn't it be useful to keep the _implies_ in the label rather than lose it?

Reply:
I think it was just a bit of wishy washiness trying not to just write verbatim Manchester syntax expressions.

Reply:
I updated to go with "present".